src/sync/app.cpp

```diff
@@ -228,6 +228,29 @@ void App::UsernamePasswordProviderClient::resend_confirmation_email(const std::s
     }, handler);
 }
 
+void App::UsernamePasswordProviderClient::retry_custom_confirmation(const std::string& email,
```

Comment:
Is this the correct function name? Maybe I'm misunderstanding, but it seems like `call_custom_confirmation_function` might be a bit clearer.

Reply:
Sounds like we need to agree on how to expose this to users 😅 https://github.com/realm/realm-java/pull/7079#discussion_r484393051

Reply:
By popular vote, we agreed on `retry_custom_confirmation`